datafusion/sql/src/planner.rs

```diff
@@ -5013,6 +5025,67 @@ mod tests {
         quick_test(sql, expected)
     }
 
+    #[test]
+    fn cte_with_no_column_names() {
+        let sql = "WITH \
+        numbers AS ( \
+            SELECT 1 as a, 2 as b, 3 as c \
+        ) \
+        SELECT * FROM numbers;";
+
+        let expected = "Projection: #numbers.a, #numbers.b, #numbers.c\
```

Comment:
Could you merge lastest from master - we no longer prefix column names with `#`

Reply:
Should be done now, thanks for reminding!